float32 dist
int32 centralPixel
